ixofoundation/ixo-webclient
# Get Outcome parameters from the Blocksync API

## Scope
[Ticket: WEB-247](https://ixo.atlassian.net/browse/WEB-247)
[Sub issue: WEB-249](https://ixo.atlassian.net/browse/WEB-249)

## Work Done
- fetch entities from blocksync api which has `Claim Type: Outcome`
- match fields on outcome targets

## Steps to Test
- filter project type investment on entities explorer page and view one of the entity
- on overview page, view FUNDING tab to see bond interface
- find Outcomes item on left handed sidebar

[Test url](https://app-uat.ixo.world/projects/did:ixo:JQmv4yyg4erjjCVuRY3g1z/bonds/did:ixo:U7GK8p8rVhJMKhBVRCJJ8c/outcomes)

## Screenshots
![image](https://user-images.githubusercontent.com/64268193/140068498-d10cc640-35ed-4e88-aa25-c7e2910438b1.png)
